openstack/db/v1/instances/testing/fixtures_test.go

```diff
@@ -104,6 +104,7 @@ var instanceGet = `
 var createReq = `
 {
 	"instance": {
+		"availability_zone": "us-east1",
```

Comment:
you need to fix alignments:

```
  	opts := instances.CreateOpts{
 		AvailabilityZone: "us-east1",
-		Name:      "json_rack_instance",
-		FlavorRef: "1",
+		Name:             "json_rack_instance",
+		FlavorRef:        "1",
 		Databases: db.BatchCreateOpts{
 			{CharSet: "utf8", Collate: "utf8_general_ci", Name: "sampledb"},
 			{Name: "nextround"},
@@ -50,8 +50,8 @@
 
 	opts := instances.CreateOpts{
 		AvailabilityZone: "us-east1",
-		Name:      "json_rack_instance",
-		FlavorRef: "1",
+		Name:             "json_rack_instance",
+		FlavorRef:        "1",
 		Databases: db.BatchCreateOpts{
 			{CharSet: "utf8", Collate: "utf8_general_ci", Name: "sampledb"},
 			{Name: "nextround"},
```

https://github.com/gophercloud/gophercloud/actions/runs/7575073607/job/20657988208?pr=2871

Reply:
Fixed: https://github.com/gophercloud/gophercloud/pull/2871/commits/3b1f30b7398fdba5cc5e9be099bafdde9b986280